wafer/talks/templatetags/review.py

```diff
@@ -0,0 +1,23 @@
+from django import template
+from django.utils.html import format_html
+from django.utils.translation import ugettext as _
+
+register = template.Library()
+
+
+@register.simple_tag()
+def reviewed_badge(user, talk):
+    """Returns a badge for the user's reviews of the talk"""
+    if user.is_anonymous():
+        return ''
+    review = talk.reviews.filter(reviewer=user).first()
+    if not review:
+        return ''
+    if review.is_current():
+        return format_html(
+            '<span class="badge badge-success" title="{}">R</span>',
```

Comment:
We already use "R" for not accepted talks. while the styling will be different, it's probably a good idea to not reuse the same letter for different status indications.

Reply:
Whoops, missed that.